src/utils/report.ts

```diff
@@ -28,10 +28,24 @@ function recordConver (args: ConverObj) {
   })
 }
 
+function _sortByNum (reportList: ConverObj[] = []) {
```

Comment:
IMO, we don't have any conventions that start with the `_` character.
```suggestion
function sortByNum (reportList: ConverObj[] = []) {
```